packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -118,6 +127,7 @@ export const createDelegationTracker = ({
     shutdown: () => {
       rewardAccounts$.complete();
       rewardsHistory$.complete();
+      logger.debug('Shutdown complete');
```

Comment:
Maybe just 'Shutdown' since this is synchronous?

```suggestion
      logger.debug('Shutdown');
```